File: ClientApp/src/app/Models/Liste_Utilisateur.ts
interface Liste_Utilisateur{
    
    civiliteAbregeUtilisateur: string,
    civiliteUtilisateur: string,
    emailUtilisateur: string,
    fonctionGeneriqueUtilisateur: string,
    fonctionUtilisateur: string,
    idResponsable: number,
    idSocieteUtilisateur: number,
    idStatutUtilisateur:number,
    idTypeUtilisateur:number,
    idUtilisateur:number,
    importanceTypeUtilisateur:number,
    loginUtilisateur:string,
    motdePasseUtilisateur:string,
    nomUtilisateur:string,
    portableUtilisateur:string,
    prenomUtilisateur:string,
    priorite:string,
    statutUtilisateur:string,
    telephoneUtilisateur:string,
    typeUtilisateur:string
   // __proto__:object,
}
interface ObjectResult_Liste_Utilisateur {
    description: string;
    errorCode: string;
    result: Liste_Utilisateur[];
}
interface ObjectResult_LU {
    description: string;
    errorCode: string;
    result: Liste_Utilisateur;
}

File: Models/Liste_Utilisateur.cs
using System;
using System.Collections.Generic;

namespace PremiumcontactMaster.Models
{
    public class Liste_Utilisateur
    {
        public int IDUtilisateur { get; set; }
        public int IDSocieteUtilisateur { get; set; }
        public string CiviliteAbregeUtilisateur { get; set; }
        public string PrenomUtilisateur { get; set; }
        public string NomUtilisateur { get; set; }
        public string LoginUtilisateur { get; set; }
        public string MotdePasseUtilisateur { get; set; }
        public string EmailUtilisateur { get; set; }
        public string TelephoneUtilisateur { get; set; }
        public int IDTypeUtilisateur { get; set; }
        public string TypeUtilisateur { get; set; }
        public byte IDStatutUtilisateur { get; set; }
        public string FonctionGeneriqueUtilisateur { get; set; }
        public int IDResponsable { get; set; }
        public string StatutUtilisateur { get; set; }
        public string FonctionUtilisateur { get; set; }
        public int? ImportanceTypeUtilisateur { get; set; }
        public string CiviliteUtilisateur { get; set; }
        public string PortableUtilisateur { get; set; }
        public string Priorite { get; set; }
    }
}
